identity/src/commonMain/kotlin/com/android/identity/nfc/NfcIsoTag.kt

```diff
@@ -224,8 +226,5 @@ abstract class NfcIsoTag {
     }
 }
 
-private fun encodeShort(value: Int): ByteArray =
-    byteArrayOf(value.shr(8).and(0xff).toByte(), value.and(0xff).toByte())
+private fun Int.encodeShort(): ByteArray = ByteArray(2).apply { putUInt16(0, toUInt() ) }
```

Comment:
I would expect this - just like for decodeShort() - would be replaced by e.g. putUin16() calls directly.